AElf.Contracts.Consensus.Tests/ConsensusContractTestAElfModule.cs

```diff
@@ -1,28 +1,50 @@
+using System;
+using System.IO;
+using AElf.Common;
+using AElf.Contracts.Genesis;
 using AElf.Contracts.TestBase;
-using AElf.Kernel.Blockchain.Application;
-using AElf.Kernel.ChainController.Application;
+using AElf.Kernel;
+using AElf.Kernel.Node;
 using AElf.Kernel.SmartContract.Application;
-using AElf.Kernel.SmartContractExecution.Application;
-using AElf.Kernel.SmartContractExecution.Domain;
+using AElf.Kernel.SmartContractExecution;
 using AElf.Modularity;
+using AElf.OS;
+using AElf.Runtime.CSharp;
+using AElf.RuntimeSetup;
+using Google.Protobuf;
 using Microsoft.Extensions.DependencyInjection;
+using Volo.Abp;
+using Volo.Abp.Autofac;
 using Volo.Abp.Modularity;
 
 namespace AElf.Contracts.Consensus.Tests
 {
     [DependsOn(
-        typeof(ContractTestAElfModule)
-    )]
+        typeof(RuntimeSetupAElfModule),
+        typeof(AbpAutofacModule),
+        typeof(SmartContractExecutionAElfModule),
+        typeof(CSharpRuntimeAElfModule2),
+        typeof(KernelAElfModule),
```

Comment:
make KernelAElfModule depends on AElf.Kernel.* modules and here only depend on Kernel module and other useful module

Reply:
Now use ContractTestAElfModule to add the dependents of other useful module.